readme.md

```diff
@@ -63,9 +63,12 @@ See [`pymatviz/ptable.py`](pymatviz/ptable.py). Heatmaps of the periodic table c
 ## Nested Periodic Table
 
 See [`pymatviz/ptable.py`](pymatviz/ptable.py). Plot histogram plots and scatter plots nested within periodic table.
-| [`ptable_hists(data, colormap="coolwarm"`](pymatviz/ptable.py) | [`ptable_scatters(data, colormap="coolwarm"`](pymatviz/ptable.py)    |
-| :------------------------------------------------------------: | :------------------------------------------------------------:    |
-|                       ![ptable_hists]                          |                       ![ptable_scatters]                          |
+| [`ptable_hists(data, colormap="coolwarm"`](pymatviz/ptable.py) | [`ptable_scatters(data, colormap="coolwarm"`](pymatviz/ptable.py) |
+| :------------------------------------------------------------: | :---------------------------------------------------------------: |
+|                        ![ptable-hists]                         |                        ![ptable-scatters]                         |
+
+[ptable-hists]: https://raw.githubusercontent.com/janosh/pymatviz/main/assets/ptable-hists.svg
+[ptable-scatters]: https://raw.githubusercontent.com/janosh/pymatviz/main/assets/ptable-scatters.svg
```

Comment:
Sorry this is an overlook from my side, forgot to replace "_" with "-". Meanwhile I thought we're putting all `assert` svgs together at the end of the `readme.md`?